jekyll/_cci2/rerun-failed-tests-only.adoc

```diff
@@ -124,10 +125,10 @@ gem 'rspec_junit_formatter'
  - run: mkdir ~/rspec
  - run:
     command: |
-      circleci tests glob spec/**/*_spec.rb | circleci tests run --command="xargs bundle exec rspec --format progress --format RspecJunitFormatter -o ~/rspec/rspec.xml" --verbose --split-by=timings
+      circleci tests glob spec/**/*_spec.rb | circleci tests run --command="xargs bundle exec rspec --format progress --format RspecJunitFormatter -o ~/rspec/rspec.xml" --verbose --split-by=timings #--split-by=timings is optional, only use if you are using CircleCI's test splitting
```

Comment:
```suggestion
      circleci tests glob "spec/**/*_spec.rb" | circleci tests run --command="xargs bundle exec rspec --format progress --format RspecJunitFormatter -o ~/rspec/rspec.xml" --verbose --split-by=timings
```
Are we seeing users include the `--split-by=timings` flag when they're not supposed to? I think it might be OK to indicate that it's optional in just the first (general) example and keep the rest of the code examples a bit cleaner. I could go either way.